vardbg/output/video_writer/config.py

```diff
@@ -106,7 +105,22 @@ def __init__(self, config_path):
         self.font_heading = (sub_path(fonts["heading"]), fonts["heading_size"])
         self.font_intro = (sub_path(fonts["intro"]), fonts["intro_size"])
 
-        style = MonokaiStyle
+        theme = self.data["theme"]
+        theme_name = theme["video_theme"]
+        # Condition for theme name having '_' 
+        if '_' in theme_name:
+            name = theme_name.split("_")
+            theme_name_style_part = "".join(word.capitalize() for word in name)
+            from_statement = 'from pygments.styles.'+theme_name + ' import '+theme_name_style_part+'Style'
+            exec(from_statement)
+            style = eval(theme_name_style_part + 'Style')
+        else:
+            from_statement = 'from pygments.styles.'+theme_name + ' import '+theme_name.capitalize()+'Style'
+            # string is executed as python code
+            exec(from_statement)
```

Comment:
`exec` and `eval` should not be used on arbitrary user input like this as it creates a code injection vulnerability. Pygments offers an API to get a style by its name directly, so this is unnecessary.

Reply:
Okay, I will look into it.